### PR TITLE
API v3: less overrides

### PR DESCRIPTION
**Pull request by stsewd**
So, conditionally setting attributes
at the class is good,
but when running tests it doesn't re-evaluate
the class, so we are stuck with the previous values.

Since no project on .org has organizations/teams
it doesn't matter exposing these fields (they return null, empty when
passed to ?expand=).

### CODE REVIEW

File: readthedocs/api/v3/permissions.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def has_permission(self, request, view):
             return True
 
 
-class PublicDetailPrivateListing(IsAuthenticated):
+class PublicDetailPrivateListing(BasePermission):
```
Comment (stsewd) [inline readthedocs/api/v3/permissions.py:20]: this class is overriding the has_permission method, and not calling super(), so it doesn't matter. We already have a check for auth users in another class

https://github.com/readthedocs/readthedocs.org/blob/4a19a9e668008fd65ee33fffe2471299ab1db492/readthedocs/api/v3/permissions.py#L76-L93

File: readthedocs/api/v3/tests/mixins.py [hunk 2]
```diff
@@ -43,6 +43,7 @@ def setUp(self):
         # objects (like a Project for translations/subprojects)
         self.project = fixture.get(
             Project,
+            id=1,
```
Comment (stsewd) [inline readthedocs/api/v3/tests/mixins.py:46]: While running apiv3 tests on .com with -k this was changed to 3, better to always be deterministic.

File: readthedocs/api/v3/views.py [hunk 3]
```diff
@@ -120,6 +120,8 @@ class ProjectsViewSetBase(APIv3Settings, NestedViewSetMixin, ProjectQuerySetMixi
         'active_versions',
         'active_versions.last_build',
         'active_versions.last_build.config',
+        'organization',
+        'teams',
```
Comment (stsewd) [inline readthedocs/api/v3/views.py:124]: teams was documented as allowed on .com, but it wasn't included on .com :D

**Review by ericholscher — APPROVED** [review]
Looks simple enough 👍

File: readthedocs/api/v3/permissions.py (hunk 1 above)
Comment (ericholscher) [reply readthedocs/api/v3/permissions.py:20]: Confirmed.

**Pull request merged**